editor/components/media-placeholder/index.js

```diff
@@ -99,22 +100,50 @@ class MediaPlaceholder extends Component {
 			onSelectUrl,
 			onHTMLDrop = noop,
 			multiple = false,
+			hasUploadPermissions,
 			additionalNotices,
 			noticeUI,
 		} = this.props;
+		const defaultInstruction = onSelectUrl ? '' : __( 'To edit this block, you need permission to upload media.' );
+		const instructions = hasUploadPermissions ? sprintf( __( 'Drag %s, upload a new one, or select a file from your library.' ), labels.name ) : defaultInstruction;
 
 		return (
 			<Placeholder
 				icon={ icon }
 				label={ labels.title }
-				instructions={ sprintf( __( 'Drag %s, upload a new one or select a file from your library.' ), labels.name ) }
+				instructions={ instructions }
 				className={ classnames( 'editor-media-placeholder', className ) }
 				notices={ <Fragment>{ additionalNotices }{ noticeUI }</Fragment> }
 			>
-				<DropZone
-					onFilesDrop={ this.onFilesUpload }
-					onHTMLDrop={ onHTMLDrop }
-				/>
+				{ hasUploadPermissions && (
+					<div className="editor-media-placeholder__upload-controls">
```

Comment:
`Placeholder` component also wraps this content with `div`:
```jsx
<div className="components-placeholder__fieldset">
	{ children }
</div>
```
What is the reason that we need another `div` wrapper?

Reply:
@imath ^ Do you have an answer to this question?

Reply:
@danielbachhuber Yes, my answer is React is throwing an error otherwise. It seems you must wrap adjacent tags into a container.

Reply:
You can use Fragment component instead. See other files you modified to check how it works - in general it groups items virtually but doesn’t render any HTML wrapper.

Reply:
Thanks a lot, i'll update the PR asap.